pkg/services/licensing/oss.go

```diff
@@ -41,7 +41,7 @@ func (l *OSSLicensingService) LicenseURL(user *models.SignedInUser) string {
 		return l.Cfg.AppSubURL + "/admin/upgrading"
 	}
 
-	return "https://grafana.com/products/enterprise/?utm_source=grafana_footer"
+	return "https://grafana.com/oss/grafana"
```

Comment:
I think we should keep the utm_source query parameter for tracking purposes.

```suggestion
	return "https://grafana.com/oss/grafana?utm_source=grafana_footer"
```

Reply:
@RichiH  @daniellee @torkelo any thoughts?

Reply:
yes, the tracking should not be removed